test/snapshots/ui.ts.crdt

```diff
@@ -1,63 +1,71 @@
-SCENE_COMPILED_JS_SIZE_PROD=295k bytes
+SCENE_COMPILED_JS_SIZE_PROD=296.77k bytes
 This run is in PROD mode.
 (start empty vm 0.21.0-3680274614.commit-1808aa1)
   OPCODES ~= 0k
   MALLOC_COUNT = 1005
   ALIVE_OBJS_DELTA ~= 0.19k
 EVAL test/snapshots/ui.js
-  REQUIRE: ~system/EngineApi
   REQUIRE: buffer
   REQUIRE: long
-  OPCODES ~= 21k
-  MALLOC_COUNT = 14265
-  ALIVE_OBJS_DELTA ~= 2.70k
+  REQUIRE: ~system/EngineApi
```

Comment:
Why double `REQUIRE: ~system/EngineApi`?

Reply:
rollup and terser cached the require(engineApi), here it is performed twice. it is semantically OK